files/keys/michaeld.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINzXuaBkXivTxDMpuF9z0voWM6mBRr5z0Laf0Y92cruG michaeld@galaxy
```

Comment:
If I remember right this is too modern for the VMs that are pre-22.04, dev for example. Do you have and rsa key? https://www.ssh.com/academy/ssh/keygen